packages/flutter/lib/src/material/expansion_panels.dart

```diff
@@ -0,0 +1,294 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'expand_icon.dart';
+import 'mergeable_material.dart';
+import 'theme.dart';
+
+const double _kPanelHeaderCollapsedHeight = 48.0;
+const double _kPanelHeaderExpandedHeight = 64.0;
+
+/// This callback gets called when [ExpansionPanel] with index [panelIndex] gets
+/// expanded or collapsed. In the case where it is expanded, [isExpanded] will
+/// be true.
+typedef void ExpansionPanelCallback(int panelIndex, bool isExpanded);
+
+/// This widget builder is useful when the header of the [ExpansionPanel] needs
+/// react to expansion.
+///
+/// An example would be to use an [AnimatedCrossfade] that triggers when
+/// [isExpanded] changes.
+typedef Widget ExpansionPanelHeaderBuilder(BuildContext context, bool isExpanded);
+
+/// A Material expansion panel. It has a header and a body and can be either
+/// expanded or collapsed. The body of the panel is only visible when it is
+/// expanded.
+///
+/// Expansion panels are used as children for [ExpansionPanelList].
```

Comment:
I assume that you mean that expansion panels can be used stand-alone as well as within ExpansionPanelLists?